systems/plants/test/testURDFmex.m

```diff
@@ -14,76 +14,117 @@
   error('Drake:MissingDependency','testURDFmex requires that urdfManipulatorDynamicsTest is built (from the command line).  skipping this test');
 end
 
-tol = 1e-2; % low tolerance because i'm writing finite precision strings to and from the ascii terminal
+tol = .1; % low tolerance because i'm writing finite precision strings to and from the ascii terminal
```

Comment:
.1 is very high. Can this really be attributed to precision of printing as a string?